@here/harp-mapview/lib/DepthPrePass.ts

```diff
@@ -131,42 +130,30 @@ export function createDepthPrePassMesh(mesh: THREE.Mesh): THREE.Mesh {
  */
 export function setDepthPrePassStencil(depthMesh: THREE.Mesh, colorMesh: THREE.Mesh) {
     // Set up depth mesh stencil logic.
-    // Set the depth pre-pass stencil bit for all processed fragments. We use `gl.ALWAYS` and not
-    // `gl.NOTEQUAL` to force all fragments to pass the stencil test and write the correct depth
-    // value.
-    depthMesh.onBeforeRender = chainCallbacks(
-        depthMesh.onBeforeRender,
-        (renderer, scene, camera, geometry, material, group) => {
-            const gl = renderer.getContext();
-            renderer.state.buffers.stencil.setTest(true);
-            renderer.state.buffers.stencil.setMask(DEPTH_PRE_PASS_STENCIL_MASK);
-            renderer.state.buffers.stencil.setOp(gl.KEEP, gl.KEEP, gl.REPLACE);
-            renderer.state.buffers.stencil.setFunc(gl.ALWAYS, 0xff, DEPTH_PRE_PASS_STENCIL_MASK);
-        }
-    );
+    // Set the depth pre-pass stencil bit for all processed fragments. We use
+    // `THREE.AlwaysStencilFunc` and not `THREE.NotEqualStencilFunc` to force all fragments to pass
+    // the stencil test and write the correct depth value.
+    (depthMesh.material as THREE.MeshStandardMaterial).stencilWrite = true;
```

Comment:
Why not get some
`const depthMeshMaterial = depthMesh.material as THREE.MeshStandardMaterial;`
as constant and use it further?

Reply:
Wrote a quick fix, but this is indeed a better approach. Will do.